### ISSUE TITLE
Update the settings locations used in the ibex_vis repo

### ISSUE THREAD

**Issue by KathrynBaker** [ISISComputingGroup/IBEX#8238]
As someone supporting the use of the official VIs in the repo under IBEX I want to not lose the settings with each upgrade so need to move the file location in use to a more appropriate directory

### Acceptance Criteria
- [x] The file paths for `setup.ini` files are all changed to somewhere appropriate in `\Instrument\Settings`

### Notes
- Need to do this before starting to use the versions of the VIs in the repo

**Comment by KathrynBaker** [comment]
The following "items" are in use:

	- [ ] HIFI Laser - not run on an instrument machine but on a separate computer to allow interaction with hardware, not altering this code
	- [ ] DDS - not run on an instrument machine but on a separate computer to allow interaction with hardware, not altering this code
	- [ ] ENGINX Positioner
	- [ ] NIMA Trough
	- [ ] Couette Cell
	- [ ] 3D Magnet
	- [ ] Muon Jaws

**Comment by KathrynBaker** [comment]
There are Agilent items which have already been replaced with more conventional IOCs, which will not be corrected here either.
The Delft code is currently not in scope - although some consideration needs to be given to this elsewhere.
The FMR code has not been included either.

**Comment by KathrynBaker** [comment]
Agilent were confirmed as only the 33220A being referenced in any configs, as such the other LVDCOM repos have been archived, and the IOCs update to aid that has been included as a pull request here.
The removal was done using `git rm <path-to-submodule>-r` to remove it from git, and then a stright `rm <path-to-submodule> -r` to delete the folders.
As it is isn't easily clear which repo is in use for the 33220A the LabVIEW VIs have been added here, and the ini file moved to settings along with the others above.

**Comment by KathrynBaker** [comment]
Pull request for tidying up the IOCs: https://github.com/ISISComputingGroup/EPICS-ioc/pull/918
Pull request for the VIs https://github.com/ISISComputingGroup/ibex_vis/pull/8

Note that https://github.com/ISISComputingGroup/IBEX/issues/8239 exists to update the LVDCOM IOCs, and until that is done these are not ready for use on beamlines, but can be used for reference by devs if needed.
